src/source/worker_source.js

```diff
@@ -9,7 +9,6 @@ import type {Bucket} from '../data/bucket';
 import type FeatureIndex from '../data/feature_index';
 import type {CollisionBoxArray} from '../data/array_types';
 import type DEMData from '../data/dem_data';
-import type {PerformanceResourceTiming} from '../types/performance_resource_timing';
```

Comment:
Why is this file being deleted?

Reply:
The type is present in the standard Flow definitions library, here: https://github.com/facebook/flow/blob/master/lib/bom.js#L250